### PR TITLE
Rainfall: try using index as home template

### PR DESCRIPTION
**Pull request by mikachan**
#### Changes proposed in this Pull Request:
This moves the contents of the home.html template to index.html, in order to try out not including a home.html template.

I've also added an archive.html template, which uses the previous index.html content.

### CODE REVIEW

File: rainfall/parts/query.html [hunk 1]
```diff
@@ -1,35 +1,51 @@
...
-<!-- /wp:column --></div>
-<!-- /wp:columns --></div>
+        <!-- wp:column {"width":"30%","style":{"spacing":{"padding":{"top":"var:preset|spacing|40","right":"var:preset|spacing|40","bottom":"var:preset|spacing|40","left":"var:preset|spacing|40"}}}} -->
+        <div class="wp-block-column" style="padding-top:var(--wp--preset--spacing--40);padding-right:var(--wp--preset--spacing--40);padding-bottom:var(--wp--preset--spacing--40);padding-left:var(--wp--preset--spacing--40);flex-basis:30%">
+            <!-- wp:template-part {"slug":"sidebar"} /-->
+        </div>
+        <!-- /wp:column -->
+    </div>
+    <!-- /wp:columns -->
+</div>
```
Comment (mikachan) [inline rainfall/parts/query.html:50]: These changes are just indenting the markup, so I could more easily see the differences between this file and home-query..

File: rainfall/patterns/home-query.php [hunk 2]
```diff
@@ -16,23 +16,24 @@
...
-<!-- wp:query-pagination {"layout":{"type":"flex","justifyContent":"space-between"}} -->
-<!-- wp:query-pagination-previous /-->
-
-<!-- wp:query-pagination-next /-->
-<!-- /wp:query-pagination --></div>
+    <!-- wp:query-pagination {"layout":{"type":"flex","justifyContent":"space-between"}} -->
+        <!-- wp:query-pagination-previous /-->
+        <!-- wp:query-pagination-next /-->
+    <!-- /wp:query-pagination -->
+</div>
```
Comment (mikachan) [inline rainfall/patterns/home-query.php:38]: These changes are just indents too.

**Review by madhusudhand — APPROVED** [review]
These changes looks good. 
While testing I have found the following minor issues.

1. Search text in the search bar is same as background, so it is not visible
image

and is it intentional to not have the border between the posts in search and archive?

2. Search/archive posts are not aligned with other content (in small screens)
image

**Comment by mikachan** [comment]
Thanks @madhusudhand!! Good spots!

> Search text in the search bar is same as background, so it is not visible

Pushed a fix here: https://github.com/Automattic/themes/pull/6701/commits/47b7760dfce835d37923fd2d9b452eda86e3fcee

> Search/archive posts are not aligned with other content (in small screens)

Pushed a fix here: https://github.com/Automattic/themes/pull/6701/commits/10e49f613b48a1d730193be6ef0a3f050835e953

> and is it intentional to not have the border between the posts in search and archive?

No this isn't intentional, this border should show. It's a bottom border on each post element (from query.html). It's showing like this on my local site:

image

I wonder why it's not showing for you!

**Comment by madhusudhand** [comment]
Thanks for addressing the issues. It looks good now.

> I wonder why it's not showing for you!

Thanks for confirming the issue. After a rebuild of Gutenberg it started working. It might be due to cache.

**Pull request merged**